PowerPointLabs/Test/Util/WPFWindowUtil.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Diagnostics;
+using System.Windows;
+using System.Windows.Threading;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using PowerPointLabs.ZoomLab.Views;
+using TestInterface;
+using TestInterface.Windows;
+
+namespace Test.Util
+{
+    static class WPFWindowUtil
+    {
+        public static string After(this string original, string searchTerm)
+        {
+            int index;
+            return (index = original.LastIndexOf(searchTerm)) != -1 ? original.Substring(index + searchTerm.Length) : "";
```

Comment:
Might it be better if the assignment to index is done on the previous line? `int index = original.LastIndexOf(searchTerm))`